server/models/ticket.js

```diff
@@ -40,7 +67,43 @@ TicketSchema.statics = {
         reject(error);
       });
     });
+  },
+
+  addAction(id, action, timestamp, userId) {
```

Comment:
Is there a use case for passing in a timestamp?

Also, since this is the only optional argument, I'd recommend moving it to the end of the arguments, so that it's more obvious.